devops/openshift/queues-deploy.yml

```diff
@@ -0,0 +1,161 @@
+    metadata:
+      name: ${NAME}
+    spec:
+      replicas: "${{REPLICAS}}"
+      revisionHistoryLimit: 10
+      selector:
+        deploymentconfig: ${NAME}
+      strategy:
+        type: Rolling
+      template:
+        metadata:
+          labels:
+            deploymentconfig: ${NAME}
+        spec:
+          containers:
+            - env:
+                - name: VERSION
+                  value: ${BUILD_TAG}
+                - name: NODE_ENV
+                  value: ${NODE_ENV}
+                - name: POSTGRES_HOST
+                  value: "${DB_SERVICE}"
+                - name: REDIS_HOST
+                  value: "${REDIS_SERVICE}"
+                - name: POSTGRES_USER
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DB_USERNAME_KEY}
+                      name: ${DB_SECRET_NAME}
+                - name: POSTGRES_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DB_PASSWORD_KEY}
+                      name: ${DB_SECRET_NAME}
+                - name: POSTGRES_DB
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DATABASE_NAME_KEY}
+                      name: ${DB_SECRET_NAME}
+                - name: REDIS_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${REDIS_PASSWORD_KEY}
+                      name: ${REDIS_SECRET_NAME}
+                - name: QUEUE_DASHBOARD_USER
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DASHBOARD_USER_KEY}
+                      name: ${QUEUES_SECRET_NAME}
+                - name: QUEUE_DASHBOARD_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DASHBOARD_PASSWORD_KEY}
+                      name: ${QUEUES_SECRET_NAME}
+                - name: ZEEBE_ADDRESS
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${ZEEBE_ADDRESS_KEY}
+                      name: ${QUEUES_SECRET_NAME}
+                - name: ZEEBE_CLIENT_ID
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${ZEEBE_CLIENT_ID_KEY}
+                      name: ${QUEUES_SECRET_NAME}
+                - name: ZEEBE_CLIENT_SECRET
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${ZEEBE_CLIENT_SECRET_KEY}
+                      name: ${QUEUES_SECRET_NAME}
+                - name: ZEEBE_AUTHORIZATION_SERVER_URL
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${ZEEBE_AUTHORIZATION_SERVER_URL_KEY}
+                      name: ${QUEUES_SECRET_NAME}
+                - name: BYPASS_CRA_INCOME_VERIFICATION
+                  value: ${BYPASS_CRA_INCOME_VERIFICATION}
+              image: ${IMAGE_REGISTRY}/${BUILD_NAMESPACE}/${IMAGE_STREAM_TAG}
+              imagePullPolicy: Always
```

Comment:
Workers and this one are created without liveness and readiness probes. We need to have a tech-debit to add it.

Reply:
They are added in the next commit. 👍